dashboard/test/models/blockly_test.rb

```diff
@@ -141,4 +141,84 @@ def assert_equal_xml(a, b)
     level.destroy
     assert_equal xml, xml2
   end
+
+  test 'localizes authored hints' do
+    test_locale = :"te-ST"
+    level_name = 'test_localize_authored_hints'
+
+    I18n.locale = test_locale
+    custom_i18n = {
+      'data' => {
+        'authored_hints' => {
+          "#{level_name}_authored_hint" => {
+            "first": "first test markdown",
+            "second": "second test markdown",
+          }
+        }
+      }
+    }
+
+    I18n.backend.store_translations test_locale, custom_i18n
+
+    level = Level.create(
+      name: level_name,
+      user_id: 0,
```

Comment:
Given that in this context we only care about the presence of the field, do you think it's still important that it be a valid ID?